rollup-sequencer/src/index.ts

```diff
@@ -52,48 +52,56 @@ async function main() {
 	await api.derive.chain.subscribeNewHeads(async (header) => {
 		const apiAt = await api.at(header.hash);
 		console.log(`block #${header.number} was authored by ${header.author}`);
+		const selectedSequencer =
+			await apiAt.query.sequencerStaking.selectedSequencer();
+
+		if (
+			selectedSequencer.isSome &&
+			u8aToHex(selectedSequencer.unwrap()).toLowerCase() ===
+				collator.address.toLowerCase()
+		) {
```

Comment:
its cosmetic but i would rather do early return to not introduce another indentation level